install.yaml

```diff
@@ -270,22 +258,17 @@
           set +a
           
           envsubst '$PRIVATE_IP' < embed.ipxe.template > embed.ipxe
-      when: not ipxeBuilt.stat.exists
-
-    - name: Generate bootloader
-      ansible.builtin.command:
-        chdir: /opt/geproxy/ipxe/src
-        cmd: make clean bin/undionly.kpxe EMBED=/opt/geproxy/pxe/embed.ipxe
-      when: not ipxeBuilt.stat.exists
-
-    - name: Copy bootloader to location
-      ansible.builtin.copy:
-        src: /opt/geproxy/ipxe/src/bin/undionly.kpxe
-        dest: /opt/geproxy/pxe
-        remote_src: yes
-      when: not ipxeBuilt.stat.exists
-
-  - name: "Build the composition"
+        creates: /opt/geproxy/pxe/embed.ipxe
```

Comment:
Adding a second make target makes more sense than doing this by abusing `include_tasks`. KISS. This also makes the changes from above no longer desired.

DRY is not something you want to achieve in cases like this since it is (1) much less legible what is happening and you want clarity over everything, and (2) oftentimes not the case that you are actually DRY-ing things up since even in this case since you still need a loop.

Reply:
Using `make clean bin/undionly.kpxe bin-x86_64...` recompiles everytime, making the command non-idempotent. Therefore each commands needs it own block. Currently it's inside it's own file, using `include_tasks`, to allow easily adding new build targets. I should probably move the targets to a variable in `defaults.yaml`. Using `include_tasks` is just a workaround against a dumb limitation of Ansible blocks.